SecurityCardManagement/README.md

```diff
@@ -15,26 +15,27 @@ Allows to manage security card.
 
 Since DeBot is a smart contract then all functions work asyncronously by design. It means that they don't return anything despite the fact that all have `returns` value in their specification. Result callback will be called later and it must have arguments defined in `returns` section.
 
-`getBlockHashes` - get H2 and H3 hashes from security card. 
-The browser should call several functions from the security card library to get H2 and H3. The workflow:
-1. get H2 using the getHashOfEncryptedPassword function;
-2. get H3 using getHashOfCommonSecret function.
+`getBlockHashes` - get H2, H3 hashes and serial number from security card.
+The browser should call `getHashes` function from the security card library to get H2, H3 and serial number.
 
-arguments: 
+arguments:
 
-	answerId: uint32  - function id of result callback
+	answerId: uint32 - function id of result callback
+	sn: uint192 - card serial number
 
-returns: 
+returns:
 
 	h2: uint256 - H2 hash
 	h3: uint256 - H3 hash
+	sn: uint192 - card serial number
```

Comment:
Why do we need to return `sn` here? `sn` is an input argument.

Reply:
It'd rather should be removed from arguments

Reply:
done